internal/schemas/locust_values.go

```diff
@@ -109,7 +117,7 @@ type LocustValues struct {
 			// SecurityGroups of ALB (separate with ,)
```

Comment:
If you don't mind, can we separate struct for nested struct?
I suggest this for future documentation of schema part :)

Reply:
Actually, securityGroups annotation in ALB use `,` for separate.
But I agree to your opinion. We can change it to struct for our values.yaml file.